tsk/fs/Makefile.am

```diff
@@ -0,0 +1,27 @@
+AM_CPPFLAGS = -I../.. -I$(srcdir)/../.. -Wall 
```

Comment:
New makefiles shouldn't be created in subdirectories. This should go into the main (and only) `Makefile.am`.

Reply:
Legacy cruft from the original PR. It's not being used. I'll delete it. Thanks for catching it.